scrapper/controllers/miner.py

```diff
@@ -3,35 +3,35 @@
 import json
 from datetime import datetime
 from bs4 import BeautifulSoup
+from flask import current_app
 
 from scrapper.models.user import User
+from scrapper import make_celery
 
+celery = make_celery(current_app)
 
-def query_twitter(user_name):
+@celery.task
+def scrape_twitter(user_name):
+    user = User()
+    user.username = user_name
     url_request = 'https://twitter.com/' + str(user_name)
     response = requests.get(url_request)
-    if response.status_code == 404:
-        return None
-
-    soup = BeautifulSoup(response.text, "html.parser")
-
-    data = soup.find("input", attrs={"class":"json-data"})
-    data = json.loads(data["value"])
-    profile_data = data["profile_user"]
-    name = profile_data["name"]
-    bio = profile_data["description"]
-    location = profile_data["location"]
-    time = datetime.now()
-    query_date = time.strftime('%m/%d/%Y %I:%M%p')
-
-    return {"username": user_name,
-            "name": name,
-            "bio": bio,
-            "location": location,
-            "query_date": query_date}
-
-
-def mine_user(user_name, refresh):
+    if not (response.status_code == 404):
```

Comment:
Are you going to try to parse a 503 also?

And also, the pythonic way for this should be `if response.status_code is not 404`. But really, you only want 200's, so forget this.